src/com/google/enterprise/adaptor/database/DatabaseAdaptor.java

```diff
@@ -357,6 +364,8 @@ static void verifyColumnNames(Connection conn, String sqlConfig, String sql,
               "Matched column \"{0}\" as \"{1}\" in query {2}",
               new Object[] { match, actual, sqlConfig });
           targets.remove(match);
+          // TODO(bmj): What happens with MySQL here.
```

Comment:
The MySQL issue is with ParameterMetaData, not ResultSetMetaData. Not to say that this couldn't blow up, and since it's optional we likely want a try-catch anyway.